src/murmur-hash/src/__tests__/murmurHash.test.js

```diff
@@ -0,0 +1,14 @@
+// @flow strict
+
+import murmurHash from '../murmurHash';
+
+it.each`
+  input                                                                          | output
+  ${'{count: 20, start: 0, end: 5}'}                                             | ${'31sjku'}
+  ${'{arg: "{arg: {count: 20, start: 0, end: 5}}"}'}                             | ${'3RGiWM'}
+  ${'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.repeat(40)} | ${'3OKbT6'}
+  ${'{}'}                                                                        | ${'2wIPj2'}
+  ${''}                                                                          | ${'0'}
+`('works as expected for output [$output]', ({ input, output }) => {
```

Comment:
```suggestion
`('works as expected for input [$input]', ({ input, output }) => {
```

Reply:
That would not work well for `'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.repeat(40)` so I added there the output so you can find the failing test case easily.